customer_outstanding_statement/__manifest__.py

```diff
@@ -1,18 +1,17 @@
-# -*- coding: utf-8 -*-
-# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
 #   (http://www.eficent.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Customer Outstanding Statement',
-    'version': '10.0.1.0.0',
+    'version': '11.0.1.0.0',
     'category': 'Accounting & Finance',
     'summary': 'OCA Financial Reports',
     'author': "Eficent, Odoo Community Association (OCA)",
     'website': 'https://github.com/OCA/account-financial-reporting',
     'license': 'AGPL-3',
     'depends': [
-        'account',
+        'account_invoicing',
```

Comment:
I prefer 'account' instead of 'account_invoicing'.

Reply:
Installing 'account' doesn't install invoicing app needed to access to the report

Reply:
Yeah, I know. But you may want to have the report installed without the module 'account_invoicing'. Maybe you have another custom module that also enables the 'invoicing' menu without using the 'account_invoicing'. 😏